vivisect/tests/stabilitydata/sudo.stability.json

```diff
@@ -16167,6 +16167,12 @@
             8,
             4,
             null
+        ],
+        [
+            35848192,
+            8,
+            4,
+            null
```

Comment:
I don't think this file should be changed either on review, I will remove it from this PR.

Reply:
???

Reply:
???

Reply:
oops, fixing